src/routes/project.route.js

```diff
@@ -2,9 +2,13 @@ const express = require("express");
 
 const router = express.Router();
 
-const verifyToken = require("./middleware/verifyToken");
 const projectController = require("./controllers/project.controller");
```

Comment:
이렇게 가져오는 방법도 있을 것 같아요~
```suggestion
const { editProject, deleteProject } = require("./controllers/project.controller");
```

Reply:
한라우터 안에 controller가 여러가지가 쓰이는 경우가있으므로 명시해주는것이 좋다고 생각합니다.

Reply:
그렇겠네요! 이해했습니다~